virt/virt-4-10-release-notes.adoc

```diff
@@ -199,7 +202,14 @@ metadata:
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
 
-//Known issues discovered after 4.9 release
+//Known issues for 4.10 release
+* `openshift-monitoring` sends an alert when an application that is protected by a Pod Disruption Budget (PDB) does not allow disruptions for migratable virtual machine images. To workaround this issue, silence the alert by setting OCP Monitoring Results.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
 
-* If a single node contains more than 50 images, pod scheduling might be imbalanced across nodes. This is because the list of images on a node is shortened to 50 by default.
-** As a workaround, you can disable the image limit by xref:../nodes/nodes/nodes-nodes-managing.adoc#nodes-nodes-managing[editing the `KubeletConfig` object] and setting the value of `nodeStatusMaxImages` to `-1`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1984442[*BZ#1984442*])
+* If the HyperConverged custom resource (CR) is configured to enable mediated devices before drivers are installed, the setting in the CR has no effect. This issue can be triggered by updates. In install order, `virt-handler` is updated before `daemonset`, which installs the NVIDIA drivers. As a result, nodes are not able to provide virtual machine GPUs.
```

Comment:
This needs some tweaking for clarity and to remove passive voice.

Reply:
* If you configure the `HyperConverged` custom resource (CR) to enable mediated devices before drivers are installed, then configuration setting changes to the CR will not be saved. This issue can be triggered by updates. For example, if `virt-handler` is updated before `daemonset`, which installs NVIDIA drivers, then nodes are not able to provide virtual machine GPUs.